EFCore.NamingConventions/NamingConventions/Internal/NamingConventionsOptionsExtension.cs

```diff
@@ -30,23 +36,23 @@ public virtual NamingConventionsOptionsExtension WithoutNaming()
             return clone;
         }
 
-        public virtual NamingConventionsOptionsExtension WithSnakeCaseNamingConvention()
+        public virtual NamingConventionsOptionsExtension WithSnakeCaseNamingConvention(CultureInfo cultureInfo = null)
```

Comment:
The parameter name here and elsewhere should be `culture`, not `cultureInfo` - this aligns with the standard .NET Core naming (e.g. [EndsWith](https://docs.microsoft.com/en-us/dotnet/api/system.string.endswith?view=netcore-3.1#System_String_EndsWith_System_String_System_StringComparison_).

Please also do the same everywhere, including for fields and properties (e.g. `_culture` instead of `_cultureInfo`).